### ISSUE TITLE
$navbar-breakpoint seems to mess with 

### ISSUE THREAD

**Issue by suterma** [jgthms/bulma#3459]
This is about **Bulma** (but it may also be lacking docs).

This is about the $navbar-breakpoint variable together with fixed and non-fixed menus.

### Overview of the problem

This is about the Bulma **CSS framework**
I'm using Bulma **version** [0.9.3]
My **browser** is: Chrome, mostly
This is a **SCSS** issue, but more about the variables
I am sure this issue is **not a duplicate** because I searched, including the closed ones.

### Description

I am playing around with the navbar top menu. I have a short menu with only 4 small entries want to have a very small breakpoint for the menu (showing the full menu on most devices, and only collapsing to the hamburger menu on very narrow screens.

In my SCSS file I have defined `$navbar-breakpoint: 320px;` before including Bulma. 

Here part of my scss files (I also use bulmaswatch)

    /* Keep the menu visible on most but the smallest devices
    * @devdoc Must be applied before the bulma.sass is used, because this is used there */
    $navbar-breakpoint: 320px;

    /** Using Bulma CSS with a theme for Bulma */
    @import '../node_modules/bulmaswatch/slate/_variables.scss';

    @import '../node_modules/bulma/bulma.sass';
    @import '../node_modules/bulmaswatch/slate/_overrides.scss';

    /* Replayer-specific (smaller) section margins */
    @import 'replayer-section.scss';

    /* Player styles (to be used for the audio player) */
    @import 'replayer-player-styles.scss';

    /* Menu styles */
    @import 'replayer-navbar-styles.scss';

   .... some more specific styles

This works nice.

However I also wanted to use the fixed menu only on the larger screens (from desktop).  I encountered the `html.has-navbar-fixed-top-desktop` style, which seems to be undocumented. **This should be documented.**

**The real problem, however, is, that the aforementioned `$navbar-breakpoint: 320px;` seems to interfere with this `html.has-navbar-fixed-top-desktop`style, setting it now also to 320px, which seems wrong.**

It's wrong because it still says "desktop" which is far from 320px, and it should also not define the fixed-menu width.

In my case I explicitly want to have a full menu on most devices, but only a fixed menu on desktop sizes. As this currently is, I can not control both independently.

I have created custom scss code to address this, which I like to share, but I guess this should be properly addressed in the framwork itself.

**Probably all there is to do is to untangle the navbar breakt point from the desktop breakpoint.**

### Steps to Reproduce

1. Define  $navbar-breakpoint: 320px;
2. Try to have a fixed menu only from the desktop width onwards

### Expected behavior

The navbar breakpoint should be as defined, AND the fixed menu should only kick in from desktop width

### Actual behavior

The fixed menu already kicks in at the navbar breakpoint

**Comment by kamatheuska** [comment]
Did you get any answer for this issue? I am having some trouble with the `.has-navbar-fixed-top` css class, and wanted to also open one.

**Comment by suterma** [comment]
No, I did not get an answer specifically to this issue. However, the fixed navbar, with the default breakpoint, works just fine as documented. So, if you can do with the default breakpoint, all should be well. If you want, you can look at how I do it in my project at https://github.com/suterma/replayer-pwa/blob/4140de5a333f80f09b2ffd6fb4b0241b9eaef558/public/index.html#L36

I have defined a custom breakpoint "laptop", to make it work with my requirments. 

I have a very small breakpoint, PLUS a fixed header on desktop and a non-fixed header on mobiles.
